docs/exclusion.md

```diff
@@ -2,44 +2,65 @@
 
 When performing validations in real world-scenarios, you may need to only run tests of a single field in your suite, or skip certain tests according to some logic. That's why Vest includes `vest.skip()` and `vest.only()`.
 
-`vest.skip()` and `vest.only()` are functions that take a name of the test, or a list of names to either include or exclude fields from being validated. They should be called from the body of `validate` callback, and in order for them to take effect, they should be called before anything else.
+`vest.skip()` and `vest.only()` are functions that take a name of the test, or a list of names to either include or exclude fields from being validated. They should be called from the body of suite callback, and in order for them to take effect, they should be called before anything else.
 
-Fields that were skipped (or not included when `only` was used) will appear in the [result object](./result) in the `skipped` array.
```

Comment:
Putting the excluded in another `skipped` path won't make my life hard as consumer. Would be glad if that can be a boolean attribute where I can get the validation in both cases (skipped / Not skipped) the same way.

Reply:
I am actually removing the skipped array in this version altogether. The `tests` object is sufficient in my opinion to know what's been tested and what's not.